templates/cisco_wlc_ssh_show_interface_detailed_id.textfsm

```diff
@@ -0,0 +1,41 @@
+Value INTERFACE_NAME (\S+)
+Value MAC_ADDRESS (\w+\.\w+\.\w+|\w+\:\w+\:\w+\:\w+\:\w+\:\w+)
+Value IP_ADDRESS (([\d1-9]+\.?){4})
```

Comment:
Do you mind changing the IP related regexes to `\d+.\d+.\d+.\d+`?

I think it's more readable that way as to what it's attempting to capture. Other than that, everything else looks good.